jsonpath_test.go

```diff
@@ -537,6 +537,92 @@ func Test_jsonpath_get_range(t *testing.T) {
 	}
 }
 
+func Test_jsonpath_get_scan(t *testing.T) {
+	obj := map[string]interface{}{
+		"key": 1,
+	}
+	res, err := get_scan(obj)
+	fmt.Println(err, res)
```

Comment:
This file seems to use both `fmt.Println` and `t.Logf` in the tests, but `t.Logf` is preferable, so let's use that here.